.github/scripts/install-slices/install_slices.py

```diff
@@ -280,6 +293,43 @@ def install_slice(slice: str, arch: str, release: str) -> None:
                 res.stderr.rstrip(),
             )
             sys.exit(res.returncode)
+        # Check if the copyright file has been installed with this slice
+        copyright_file = pathlib.Path(f"{tmpfs}/usr/share/doc/{pkg}/copyright")
+        installed_copyright[slice_name] = (
+            copyright_file.is_file() or copyright_file.is_symlink()
+        )
+
+
+def deb_has_copyright_file(pkg: str) -> bool:
+    """
+    Checks if a deb's contents comprise a copyright file
+
+    NOTE: this is a temporary and convoluted implementation, as at the moment
+    we don't have an easy and reliable way to check which deb was used for
+    the installation (at least not without duplicating a some of the Chisel
+    codebase).
+
+    TODO: update this function once the Chisel DB is available, as the pkg
+    SHAs will be available from the DB itself.
+    """
+    pkg_shas = os.popen(
+        f"grep -A 25 -R 'Package: {pkg}$' {str(CHISEL_PKG_CACHE)} | grep SHA256 | cut -d ' ' -f 2"
+    ).read()
+    for deb in pkg_shas.splitlines():
```

Comment:
What do you think about iterating over all the files and checking (a) if the file is a Debian package and (b) if so, if it is the package we are looking for?

I think this `grep` here might slow the script down a bit. For example, this command took 33 seconds in my local machine:

```
time grep -A 25 -R "Package: base-files" ~/.cache/chisel/sha256  | grep SHA256 | cut -d ' ' -f 2
...
real	0m33.735s
user	0m7.389s
sys	0m7.526s
```

Reply:
I think this might be less of a problem if the cache is almost empty which might be the case for the CI (not sure though).

Reply:
should be improved by a factor of ~2x in 3f1fbac